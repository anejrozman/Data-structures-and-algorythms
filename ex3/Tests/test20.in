283,282,59
6,2,9,8,5,1,5,2,4,10,10,7,6,4,2,3,8,1,7,10,3,5,9,7,10,10,1,2,1,7,3,3,5,7,7,9,2,5,5,2,2,2,6,9,9,8,9,8,3,2,4,8,9,6,6,10,9,5,5,6,10,3,2,6,5,10,10,6,4,9,8,8,1,1,2,2,2,5,5,10,4,1,7,3,1,1,5,7,1,7,8,3,8,2,8,4,10,9,8,9,7,4,3,3,10,8,1,2,1,10,7,4,1,2,6,7,5,7,9,9,5,10,4,8,5,2,9,1,3,3,8,9,9,8,10,2,8,1,3,7,10,9,9,5,1,4,3,10,10,6,10,10,4,9,7,10,6,10,2,7,8,4,9,2,1,9,6,2,2,7,9,3,1,3,5,5,8,4,1,1,5,9,8,1,8,7,3,1,7,7,7,2,3,5,7,3,6,4,9,3,8,2,7,8,10,7,3,9,8,6,4,1,8,3,4,10,3,10,6,6,1,3,7,10,1,1,1,9,10,2,1,2,2,1,8,5,9,7,9,8,4,9,1,2,5,6,2,5,1,7,1,6,6,6,8,3,6,8,5,6,8,1,7,3,4,6,4,10,5,1,6,6,10,4,8,8,1,8,6,8,5,5,9
131,269,2.862
274,98,515.87
111,56,2.275
207,122,5.911
82,42,524.923
42,200,529.323
16,150,18.978
254,32,17.382
50,118,535.543
187,150,538.876
193,244,27.25
231,15,29.137
130,240,543.163
270,1,31.467
125,37,543.218
261,234,548.938
153,19,37.678
268,246,550.916
19,155,549.703
48,85,562.566
141,13,51.167
153,169,51.038
38,74,53.517
196,204,564.795
54,9,566.622
109,54,56.454
89,78,56.225
1,7,65.751
139,105,69.35
239,202,69.899
165,1,586.358
190,224,75.644
261,197,588.954
212,123,586.506
171,30,82.29
155,30,84.231
45,119,85.907
220,102,84.953
120,10,598.72
187,11,88.724
234,144,602.441
266,229,605.212
165,262,94.026
230,179,607.732
22,256,611.404
262,125,104.887
91,155,616.455
29,177,105.398
160,131,104.586
227,180,620.185
159,197,110.569
111,64,623.822
19,160,624.663
150,276,625.176
193,274,112.064
154,33,626.214
238,213,629.021
96,218,117.848
76,173,119.007
3,97,634.625
118,201,124.198
145,226,125.275
18,9,126.662
88,29,636.313
195,170,128.137
4,178,129.952
250,75,643.652
133,116,647.292
211,81,648.735
274,233,648.356
72,94,137.62
61,116,138.182
247,41,135.76
219,38,137.335
80,265,654.834
63,53,655.488
49,6,656.601
21,176,655.289
222,16,655.656
35,250,144.642
187,133,148.363
273,241,148.975
262,2,655.51
189,190,151.563
128,206,152.579
24,25,661.075
54,111,155.861
189,88,669.294
34,99,158.969
261,239,670.686
92,223,158.335
263,21,165.998
66,182,170.027
132,182,175.805
119,131,177.579
69,171,690.234
222,14,691.036
208,9,180.327
123,57,692.207
105,217,694.079
142,61,181.74
205,69,698.732
162,134,188.577
62,194,700.399
90,63,703.308
247,237,704.673
56,50,195.671
81,179,709.779
68,106,197.116
49,216,711.425
26,241,714.1
154,17,714.425
169,88,204.196
222,139,718.936
84,210,719.215
143,177,208.281
145,64,143.7
125,11,720.775
247,181,724.54
145,20,724.216
172,233,726.59
165,76,215.81
143,105,215.007
191,114,729.82
152,7,220.801
230,143,220.913
106,56,222.308
187,258,735.967
210,80,735.936
91,80,225.078
35,31,738.727
174,168,227.917
97,27,734.224
149,119,229.406
169,202,229.214
202,225,734.876
96,95,750.911
154,72,239.266
180,0,750.915
271,67,244.157
93,259,757.703
3,121,246.165
163,169,247.272
22,15,761.326
271,124,250.144
235,241,251.551
165,58,253.143
26,48,766.384
42,31,255.836
277,249,257.539
70,209,259.008
109,46,260.661
127,272,774.002
265,37,262.514
63,144,263.121
140,10,266.186
211,231,268.311
120,226,269.809
219,247,784.286
201,27,790.976
141,32,795.891
254,132,796.924
53,82,798.24
24,87,288.875
260,114,801.991
239,105,292.877
14,246,293.376
78,74,806.868
92,139,296.683
218,91,297.981
37,282,809.56
90,275,812.602
12,83,301.137
24,221,302.049
106,19,815.545
251,175,821.155
236,90,825.582
174,225,317.07
146,136,830.221
188,137,318.594
235,150,319.122
158,92,321.954
94,91,834.175
235,10,833.014
85,214,832.8
84,16,324.713
163,157,838.274
161,190,840.42
87,199,331.672
217,230,332.918
107,28,333.514
236,82,846.542
228,247,331.707
197,218,848.586
219,197,336.466
245,274,849.147
17,110,853.579
93,243,343.741
180,37,343.043
69,142,345.738
188,237,351.735
250,237,865.046
243,49,865.322
184,241,355.06
235,184,872.027
164,39,365.136
263,108,366.145
257,106,879.508
135,2,366.489
204,235,372.045
165,252,377.476
13,123,378.595
128,236,890.207
23,38,890.608
269,138,893.225
78,46,381.236
158,25,383.106
111,164,384.498
170,168,377.108
243,129,377.563
168,263,899.001
141,14,900.36
67,125,901.862
172,264,392.445
206,138,906.265
25,234,908.307
181,226,909.96
188,16,914.25
17,253,407.563
165,9,920.731
161,3,920.42
174,94,410.833
66,84,923.63
255,190,924.39
207,233,923.137
29,168,412.15
3,79,415.06
104,32,928.905
151,210,928.393
234,1,409.885
239,203,419.203
117,93,932.129
202,242,421.8
246,248,934.536
152,154,935.484
100,216,420.24
227,1,937.672
11,21,426.554
106,179,937.447
80,104,424.853
71,255,947.461
35,239,950.815
160,80,439.501
262,112,440.07
14,232,952.003
232,148,957.304
119,1,957.314
236,239,959.027
58,235,960.78
276,52,960.561
108,205,962.095
116,0,450.839
170,164,964.257
204,203,450.978
212,71,967.246
152,95,968.989
281,85,973.589
110,190,462.56
63,82,980.686
140,136,981.792
114,27,981.922
271,171,987.123
42,21,989.537
187,229,991.246
162,193,482.873
179,237,483.293
188,263,482.559
260,257,482.144
53,56,486.628
13,264,487.695
119,42,496.612
214,125,500.272
