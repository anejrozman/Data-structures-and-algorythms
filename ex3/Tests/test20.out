204,642315
8,380
2,63
2,56
2,48
2,32
2,30
2,27
2,24
2,18
2,14
2,10
2,8
2,5
2,3
2,2
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
1,0
